content/en/logs/archives/s3.md

```diff
@@ -96,17 +55,53 @@ Next, grant Datadog permissions to write log archives to your S3 bucket. You can
 }
 ```
 
```

Comment:
The code block has to be indented 4 spaces so it doesn't break the numbered list (see the preview).